app/Http/Requests/CommandCenterShiftRequest.php

```diff
@@ -29,14 +29,18 @@ public function rules()
         return [
             'name' => [
                 'required',
-                Rule::unique('command_center_shifts')->ignore($this->command_center_shift),
+                Rule::unique('command_center_shifts')->ignore($this->command_center_shift)->where(function ($query) {
```

Comment:
```suggestion
                Rule::unique('command_center_shifts')->ignore($this->command_center_shift)->whereNull('deleted_at')
```
coba gini kang @iprasetyaa

Reply:
Saya coba dulu di local kang

Reply:
Kang sudah diperbaiki @ayocodingit https://github.com/jabardigitalservice/reservasi-backend/pull/96/commits/0c8a174da1f126eea75a85eeb694db57c7f43adf

Reply:
sip lebih clean :+1: